packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -14,11 +14,13 @@ import {
   TimeSeriesValue,
   FieldDTO,
   DataFrameDTO,
+  AbsoluteTimeRange,
 } from '../types/index';
 import { isDateTime } from '../datetime/moment_wrapper';
 import { ArrayVector } from '../vector/ArrayVector';
 import { MutableDataFrame } from './MutableDataFrame';
 import { SortedVector } from '../vector/SortedVector';
+import { reduceField, ReducerID } from '../transformations';
```

Comment:
the error you are getting in tests when importing toDataFrame is due to this import, think it's some recursive import that is going on but did not manage to track it down :(